### ISSUE TITLE
Spaces in image name break operation

### ISSUE THREAD

**Issue by bartmanus** [sciurus/vagrant-mutate#79]
Microsoft [offers](https://dev.windows.com/en-us/microsoft-edge/tools/vms/linux/) some Vagrant boxes for testing their browsers. They seem to have spaces in the names of the disk images which causes a failure when mutating the image to libvirt's format:
```
$ cat /etc/redhat-release 
Fedora release 23 (Twenty Three)
$ vagrant plugin list
...
vagrant-mutate (1.0.3)

$ vagrant mutate ./MSEdge - Win10.box libvirt
Extracting box file to a temporary directory.
qemu-img: Expecting one image file name
Try 'qemu-img --help' for more information
```
It seems that vagrant-mutate invokes `qemu-img` without quoting the disk image's filename. It prints the same error when invoked via bash manually without escaping spaces.

I worked around the problem by extracting the box-file (found it to be a simple `file`), renaming the image file and updating the manifest files and repackaging the files back, then it works.

**Comment by sciurus** [comment]
Thanks for reporting your experience with vagrant-mutate! I agree this is a problem and should have a fix shortly.

**Comment by sciurus** [comment]
Fix released in 1.0.4.
